### PR TITLE
cmatrix: update to 2.0

### PR DESCRIPTION
**Pull request by i0ntempest**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.15.4 19E266
Xcode 11.4 11E146

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: games/cmatrix/Portfile [hunk 1]
```diff
@@ -11,32 +11,31 @@ description			Console Matrix
...
+checksums			rmd160  73e38f7f1e903a738e98724bcc326b4c71034b44 \
+					sha256  456d8097cb4724a69337e9803dd9b7079fa10684d22e35fc2bdd88655c03925a \
+					size    205672
 
+depends_build       port:autoconf port:automake
 depends_lib         port:ncurses
 
-patchfiles          Makefile.in.patch
-
+configure.cmd       autoreconf -i && ./configure
```
Comment (reneeotten) [inline games/cmatrix/Portfile:21]: you probably want to use `use_autoreconf` here, see [the guide](https://guide.macports.org/#reference.phases.configure.automake-autoconf)

File: games/cmatrix/Portfile [hunk 2]
```diff
@@ -1,42 +1,42 @@
...
-license				GPL
-maintainers			nomaintainer
-description			Console Matrix
-long_description	Console Matrix simulates the display from "The Matrix"
-homepage			http://www.asty.org/cmatrix/
+github.setup        abishekvashok cmatrix 2.0 v
+revision            0
+categories          games x11
+platforms           darwin
+license             GPL
```
Comment (reneeotten) [inline games/cmatrix/Portfile:8]: upstream lists this as `GPL-3`

File: games/cmatrix/Portfile [hunk 3]
```diff
@@ -1,42 +1,42 @@
...
+maintainers         nomaintainer
+description         Console Matrix
+long_description    Console Matrix simulates the display from "The Matrix"
+homepage            http://www.asty.org/cmatrix/
```
Comment (reneeotten) [inline games/cmatrix/Portfile:12]: please use `https`

File: games/cmatrix/Portfile [hunk 4]
```diff
@@ -1,42 +1,42 @@
-PortSystem			1.0
```
Comment (reneeotten) [inline games/cmatrix/Portfile:1]: please add [a modeline ](https://guide.macports.org/chunked/development.creating-portfile.html)

**Comment by i0ntempest** [comment]
@reneeotten Should be good now, pls check

**Comment by reneeotten** [comment]
thanks @i0ntempest - merging this

**Pull request merged**